(docs) #4768 Add schema examples for migrations

oznu opened the pull request:
Added examples to the documentation to show how to use custom schema when executing migrations.

Resolved #4768

sushantdhiman commented on docs/docs/migrations.md:
````diff
@@ -203,14 +204,32 @@ queryInterface.addColumn(
 
 ```js
-queryInterface.removeColumn('Person', 'signature')
+queryInterface.removeColumn('nameOfAnExistingTable', 'nameOfAnExistingAttribute')
````
Don't change it to `nameOfAnExistingTable` and `nameOfAnExistingAttribute`, Its already clear from method signature. Same goes for schema example below

sushantdhiman commented:
Ping @oznu

sushantdhiman commented on docs/docs/migrations.md:
```diff
@@ -203,14 +204,32 @@ queryInterface.addColumn(
+
+queryInterface.removeColumn({
+  tableName: 'Person',
+  schema: 'nameOfAnExistingSchema'
```
Name of scheme need change too

sushantdhiman commented on docs/docs/migrations.md:
```diff
@@ -203,14 +204,32 @@ queryInterface.addColumn(
+// or with an explicit schema:
+
+queryInterface.addColumn({
+    tableName: 'nameOfAnExistingTable',
```
You forgot to change these. Use a real world example here for tableName, schema and attribute

sushantdhiman commented on docs/docs/migrations.md:
```diff
@@ -119,8 +119,9 @@ queryInterface.createTable(
-    charset: 'latin1' // default: null
+    engine: 'MYISAM',                     // default: 'InnoDB'
+    charset: 'latin1',                    // default: null
+    schema: 'nameOfTheExistingSchema'     // PostgreSQL only. default: public
```
change this too

oznu commented:
Thanks, I updated the examples I added to use real world examples. 

I can update the existing migration examples to use real work examples as well if you want, though that might be for a seperate pull request.
